src/main/java/de/orchound/gameoflife/game/Board.java

```diff
@@ -47,11 +47,9 @@ private int countLivingNeighbors(int rowIndex, int cellIndex) {
 		for (int xOffset = -1; xOffset <= 1; xOffset++) {
 			for (int yOffset = - 1; yOffset <= 1; yOffset++) {
 				if (!(xOffset == 0 && yOffset == 0)) {
-					int neighborX = cellIndex + xOffset;
-					int neighborY = rowIndex + yOffset;
-					if (neighborX >= 0 && neighborX < width && neighborY >= 0 && neighborY < height) {
-						livingNeighborsCount += source[neighborY][neighborX] ? 1 : 0;
-					}
+					int neighborX = (cellIndex + xOffset + width) % width;
+					int neighborY = (rowIndex + yOffset + width) % height;
```

Comment:
You can use `Math.floor()` to get rid of this weird "+ width". Also in line 51 it should've really been "+ height".
```suggestion
					int neighborX = Math.floorMod(cellIndex + xOffset, width);
					int neighborY = Math.floorMod(rowIndex + yOffset, height);
```